kani-compiler/src/kani_middle/points_to/points_to_analysis.rs

```diff
@@ -677,8 +675,6 @@ fn is_identity_aliasing_intrinsic(intrinsic: Intrinsic) -> bool {
         | Intrinsic::PtrOffsetFromUnsigned
         | Intrinsic::RawEq
         | Intrinsic::RetagBoxToRaw
-        | Intrinsic::RintF32
-        | Intrinsic::RintF64
```

Comment:
Should the two new intrinsics be added here?

Reply:
yes. Side note: we don't have any test that covers this